frontend/sw360-portlet/src/main/webapp/html/vulnerabilities/view.jsp

```diff
@@ -34,9 +45,61 @@
 <div id="header"></div>
 <p class="pageHeader">
     <span class="pageHeaderBigSpan">Vulnerabilities</span>
+    <span class="pageHeaderMediumSpan">(<core_rt:if test="${vulnerabilityList.size() == totalRows}">${totalRows}</core_rt:if><core_rt:if test="${vulnerabilityList.size() != totalRows}">${vulnerabilityList.size()} latest of ${totalRows}</core_rt:if>)</span>
 </p>
 <div id="searchInput" class="content1">
     <%@ include file="/html/utils/includes/quickfilter.jspf" %>
+
+    <form action="<%=applyFiltersURL%>" method="post">
+        <table>
+            <thead>
+            <tr>
+                <th class="infoheading">
+                    Loading
+                </th>
+            </tr>
+            </thead>
+            <tbody style="background-color: #f8f7f7; border: none;">
+            <tr>
+                <td>
+                    <select class="searchbar" id="view_size" name="<portlet:namespace/><%=PortalConstants.VIEW_SIZE%>" onchange="reloadViewSize()">
+                        <option value="50" <core_rt:if test="${viewSize == 50}">selected</core_rt:if>>50 latest</option>
```

Comment:
I think we can remove that option (later).

Reply:
done